packages/framework/fluid-static/api-report/fluid-static.public.api.md

```diff
@@ -43,6 +44,7 @@ export interface IFluidContainerEvents extends IEvent {
     (event: "saved", listener: () => void): void;
     (event: "dirty", listener: () => void): void;
     (event: "disposed", listener: (error?: ICriticalContainerError) => void): any;
+    (event: "readonlyChanged", listener: (readonly: boolean) => void): void;
```

Comment:
Should match camel casing between the event and property - either isReadOnly/readOnlyChanged or isReadonly/readonlyChanged.  Otherwise this naming and use of method looks good to me!